ci/release_notes.md

```diff
@@ -0,0 +1,6 @@
+# Fixes
+- acceptance tests are now run in parallel
+
+# Acknowledgements
+
+Thanks @peterellisjones for the PR
```

Comment:
Do we need a release for this? We don't touch any of the actual coding therefore I wouldn't create one.

Reply:
good point, removed